apps/guardian-ui/src/components/BalanceCard.tsx

```diff
@@ -10,9 +10,12 @@ export const BalanceCard: React.FC = () => {
   const { api } = useAdminContext();
   const [auditSummary, setAuditSummary] = useState<AuditSummary>();
 
+  // FIXME: we shouldn't default to 0 balance
   const walletBalance = auditSummary
-    ? auditSummary.module_summaries.wallet?.net_assets || (0 as MSats)
-    : undefined;
+    ? Object.entries(auditSummary.module_summaries).find(
+        (m) => m[1].kind === ModuleKind.Wallet
+      )?.[1].net_assets
+    : (0 as MSats);
```

Comment:
We really shouldn't show incorrect balances, ever. If we don't get the data we expect, I think we should display an error instead.